TrenDAP/wwwroot/TypeScript/Features/DataSets/Tabs/DataSetSettingsTab.tsx

```diff
@@ -49,9 +50,20 @@ const DataSetSettingsTab: React.FunctionComponent<IProps> = (props: IProps) => {
     }, [dataSetStatus]);
 
     function valid(field: keyof (TrenDAP.iDataSet)): boolean {
-        if (field == 'Name')
-            return props.DataSet.Name != null && props.DataSet.Name.trim().length > 0 &&
-                props.DataSet.Name.length <= 200 && dataSets.find(ws => ws.Name.toLowerCase() == props.DataSet.Name.toLowerCase() && ws.ID != props.DataSet.ID) == null
+        if (field == 'Name') {
+            if (props.DataSet.Name == null || props.DataSet.Name.trim().length == 0 || props.DataSet.Name.length > 200) {
+                setNameFeedback("Name less than 200 Characters is required.")
+                return false;
+            }
+            else if (dataSets.findIndex(ds => (ds.Public || ds.User === userName) && ds.ID !== props.DataSet.ID && ds.Name.toLowerCase() == props.DataSet.Name.toLowerCase()) > -1) {
+                setNameFeedback("A DataSet with this name already exists.")
```

Comment:
```suggestion
                setNameFeedback("A Data Set with this name already exists.")
```

make sure you update all instances to `Data Set` with a space

Reply:
Changed it everywhere in the UI I could spot